src/main/java/com/eternalcode/lobbyheads/configuration/implementation/HeadsConfiguration.java

```diff
@@ -0,0 +1,91 @@
+package com.eternalcode.lobbyheads.configuration.implementation;
+
+import com.eternalcode.lobbyheads.delay.DelaySettings;
+import eu.okaeri.configs.OkaeriConfig;
+import eu.okaeri.configs.annotation.Comment;
+import eu.okaeri.configs.annotation.Header;
+import eu.okaeri.configs.annotation.NameModifier;
+import eu.okaeri.configs.annotation.NameStrategy;
+import eu.okaeri.configs.annotation.Names;
+import org.bukkit.Particle;
+import org.bukkit.Sound;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+@Names(strategy = NameStrategy.HYPHEN_CASE, modifier = NameModifier.TO_LOWER_CASE)
+@Header("# ")
+@Header("# LobbyHeads configuration file")
+@Header("# Permissions:")
+@Header("# - lobbyheads.replace - allows replacing heads")
+@Header("# - lobbyheads.admin - allows adding/removing head")
+@Header("# ")
+public class HeadsConfiguration extends OkaeriConfig implements DelaySettings {
+
+    @Comment("# Delay between replacing heads")
+    public Duration delay = Duration.ofSeconds(15);
+
+    @Comment({ " ", "# Heads list, don't touch this!" })
+    public List<com.eternalcode.lobbyheads.head.Head> heads = new ArrayList<>();
```

Comment:
I think we can use import this class. 
```suggestion
    public List<Head> heads = new ArrayList<>();
```